docs/integrations.md

```diff
@@ -0,0 +1,38 @@
+# Integrations
+
+This project has the following integrations:
+
+* [Slack](#slack)
+* [Workplace](#workplace)
+
+## Slack
+
+Slack is a collaboration chat tool often used in and out of organizations to help team communicate and coordinate in a more effective manner.
+
+FAQ has the following integration with slack:
+
+* Sends new questions into a dedicated slack channel
+
+### Configuration
+
+In order to integrate with Slack, you will need to create an application. It can easily be done using the [Slack "Build" dashboard](https://api.slack.com/). Follow the instructions from Slack to create your app. See below for configuration for your environment variables.
```

Comment:
You can drop "easily", it's relative. Easy for someone might be easy for others.